internal/impl/gcp/bigquery.go

```diff
@@ -130,3 +132,25 @@ func errorFromStatus(status *bigquery.JobStatus) error {
 
 	return fmt.Errorf("failed to complete bigquery job successfully: %w", bqErr)
 }
+
+func parseQueryPriority(config *service.ParsedConfig, fieldName string) (bigquery.QueryPriority, error) {
```

Comment:
I extracted this into a function so I can unit test it (tests coming later today).